### PR TITLE
Add a building fuzz targets document.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL. This is loosely based on https://google.github.io/clusterfuzz/setting-up-fuzzing/libfuzzer-and-afl/, but tries to be slightly more generic.

### CODE REVIEW

File: docs/building-fuzz-targets.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+# Building fuzz targets
+
+This document focuses on the specifics of how to build fuzzers in common
+configurations useful for detecting security bugs. If you haven’t written your
```
Comment (inferno-chromium) [inline docs/building-fuzz-targets.md:4]: security and stability

**Review by inferno-chromium — APPROVED** [review]

File: docs/building-fuzz-targets.md (hunk 1 above)
Comment (mbarbella-chromium) [reply docs/building-fuzz-targets.md]: Done!

**Review by 1Elle1111 — COMMENTED** [review]
Hi

**Review by 1Elle1111 — COMMENTED** [review]
Hi

**Pull request merged**